### PR TITLE
Update multiple UI elements based on user feedback.

### PR DESCRIPTION
**Pull request by TreyE**
https://www.pivotaltracker.com/story/show/185102119
https://www.pivotaltracker.com/story/show/185102807
https://www.pivotaltracker.com/story/show/185102847
https://www.pivotaltracker.com/story/show/185235742

### CODE REVIEW

File: libs/carrier-portal/types/src/lib/person-view-data.ts [hunk 1]
```diff
@@ -135,6 +136,7 @@ export const defaultPolicy: Policy = {
     hios_id: '',
     name: '',
     product_kind: '',
+    coverage_year: '2022',
```
Comment (nisanthyaganti9) [inline libs/carrier-portal/types/src/lib/person-view-data.ts]: can you update this to empty string? it's giving false sense that we're hard coding year here!

File: libs/carrier-portal/ui/src/lib/pipes/phone-number.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      const top = number.slice(0, 2);
+      const rest = number.slice(3, 6);
+      return `${top}-${rest}`;
+    }
+    if (number.length === 10) {
+      const ac = number.slice(0, 2);
+      const top = number.slice(3, 5);
+      const rest = number.slice(6, 9);
+      return `(${ac}) ${top}-${rest}`;
+    }
```
Comment (nisanthyaganti9) [inline libs/carrier-portal/ui/src/lib/pipes/phone-number.pipe.ts:23]: Is it worth to also format phone number when it has country code in it?

File: libs/carrier-portal/types/src/lib/person-view-data.ts (hunk 1 above)
Comment (TreyE) [reply libs/carrier-portal/types/src/lib/person-view-data.ts]: Makes sense, will do.

File: libs/carrier-portal/ui/src/lib/pipes/phone-number.pipe.ts (hunk 2 above)
Comment (TreyE) [reply libs/carrier-portal/ui/src/lib/pipes/phone-number.pipe.ts:23]: Hm...I don't know if we include those let me research.

Comment (TreyE) [reply libs/carrier-portal/ui/src/lib/pipes/phone-number.pipe.ts:23]: I've added handling for those with country codes.

**Review by nisanthyaganti9 — APPROVED** [review]

**Review by kristinmerbach — APPROVED** [review]

**Pull request merged**